packages/MarkdownEditor-Core.package/MarkdownCommentBlock.class/instance/commentEnd.st

```diff
@@ -0,0 +1,6 @@
+accessing
+commentEnd
+
+	^ self isClosed
+		ifTrue: [(self content findString: '-->') + 2]
```

Comment:
The same with the 2.

Reply:
Here it could be '-->' size -1. I don't know if all this index-shifting makes it more clearer